sift/client.py

```diff
@@ -313,6 +314,135 @@ def get_workflow_status(self, run_id, timeout=None):
         except requests.exceptions.RequestException as e:
             raise ApiException(str(e))
 
+    def get_decisions(self, entity_type, limit=None, start_from=None, abuse_types=None, timeout=None):
+        """Get decisions available to customer
+
+        Args:
+            entity_type: only return decisions applicable to entity type {USER|ORDER}
+            limit: number of query results (decisions) to return [default: 100]
+            start_from: result set offset for use in pagination [default: 0]
+            abuse_types: csv of abuse_types to filter returned decisions by (optional)
+
+        Returns:
+            A sift.client.Response object containing array of decisions if call succeeded
+            Otherwise raises an ApiException
+        """
+
+        if timeout is None:
+            timeout = self.timeout
+
+        params = {}
+
+        if not isinstance(entity_type, self.UNICODE_STRING) or len(entity_type.strip()) == 0\
+                or entity_type not in {'user', 'order'}:
```

Comment:
Also, noticed that here we are validating the argument, but in `apply_order_decision()` we are apparently relying on the server to do validation for us.  Please look through the existing client code to see what course we're taking w.r.t. validating arguments on client vs server -- and make this consistent.